tests/test_prediction.py

```diff
@@ -228,3 +236,5 @@ def test_mixed_pred_upload(model_run):
     assert response["model_run_id"] == model_run.model_run_id
     assert response["predictions_processed"] == 3
     assert response["predictions_ignored"] == 0
+
+    response_refloc = model_run.refloc(prediction_polygon.reference_id)
```

Comment:
Are there supposed to be tests on this object as well

Reply:
yes thanks for catching